apps/src/templates/teacherDashboard/teacherSectionsRedux.js

```diff
@@ -218,6 +307,65 @@ export default function teacherSections(state=initialState, action) {
     };
   }
 
+  if (action.type === EDIT_SECTION_BEGIN) {
+    const initialSectionData = action.sectionId ?
+      {...state.sections[action.sectionId]} :
+      newSectionData(-1, action.courseId, undefined);
```

Comment:
Should this -1 be a constant?

Previously we could theoretically have multiple new sections in progress at once (so we would count down from -1 for ids). I think in the new design we'll only ever have one new section in progress, so we can presumably just have something like `PENDING_NEW_SECTION_ID = -1`